resources/lib/omdb/mapping.py

```diff
@@ -39,7 +39,7 @@ def __init__(self):
                 'func': lambda v: get_between_strings(v or '', 'Nominated for ', ' Golden Globe')}, {
                 # ---
                 'keys': [('infoproperties', 'award_nominations')],
-                'func': lambda v: get_between_strings(v or '', 'wins & ', ' nominations') or get_between_strings(v or '', '', ' nominations')
+                'func': lambda v: get_between_strings(v or '', 'wins & ', ' nomination') or get_between_strings(v or '', 'win & ', ' nomination') or get_between_strings(v or '', '', ' nomination')
```

Comment:
Can simplify with one or none `?` quantifier on the plural
`get_between_strings(v or '', 'wins? & ', ' nomination') or get_between_strings(v or '', '', ' nomination')`